catalog/plugin/catalog-plugin-clavin/README.md

```diff
@@ -0,0 +1,34 @@
+
+## Clavin Plugin
+
+Unforturnately, clavin is a memory hog. Before ddf startup, edit ../bin/setenv and change memory to 6 gigs:
```

Comment:
Probably best to refrain from editorializing in READMEs.  Also, typo in `Unforturnately`.